astropy_helpers/openmp_helpers.py

```diff
@@ -39,69 +41,222 @@
 }
 """
 
+def _get_flag_value_from_var(flag, var, delim=' '):
+    """
+    Utility to extract `flag` value from `os.environ[`var`]` or, if not present,
+    from `distutils.sysconfig.get_config_var(`var`)`.
+    E.g. to get include path: _get_flag_value_from_var('-I', 'CFLAGS')
+    might return "/usr/local/include".
```

Comment:
for which OS? Please keep comments it OS agnostic

Reply:
This is why I added the "E.g." and "might".